sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/FileFormatWriter.scala

```diff
@@ -217,8 +217,12 @@ object FileFormatWriter extends Logging {
 
       val commitMsgs = ret.map(_.commitMsg)
 
+      val commitJobStartTime = System.nanoTime()
+      logInfo(s"Start to commit write Job ${description.uuid}.")
       committer.commitJob(job, commitMsgs)
-      logInfo(s"Write Job ${description.uuid} committed.")
+      val commitJobEndTime = System.nanoTime()
+      logInfo(s"Write Job ${description.uuid} committed. " +
```

Comment:
How about use ` val (_, duration) = Utils.timeTakenMs { committer.commitJob(job, commitMsgs) }`

Reply:
+1 for @yaooqinn 's suggestion to use `Utils.timeTakenMs`.

Reply:
Followed this.